Frends.Confluence.Request/Frends.Confluence.Request.Tests/Tests.cs

```diff
@@ -155,4 +156,148 @@
+                SpaceId = WorkSpaceId,
+                Title = "TestPage3",
+                Body = "This is a new page 3 created by the API"
+            },
+            CancellationToken.None
+        );
+        Assert.AreEqual(200, result.StatusCode);
+    }
+
+    [TestMethod]
+    public async Task UpdatePage()
+    {
+        var result = await Confluence.UpdatePage(
+            new Input
+            {
+                Title = "UpdatedTitle",
+                PageId = PageId,
+                Username = username,
+                ApiToken = apiToken,
+                ApiVersion = ApiVersion.V2,
+                ConfluenceDomainName = domainName,
+                Body = "Updated new page created by the API",
+                Version = 2
+            },
+            CancellationToken.None
+        );
+        Assert.AreEqual(200, result.StatusCode);
+    }
+
+    [TestMethod]
+    public async Task GetPageById()
+    {
+        var result = await Confluence.GetPageById(
+            new Input
+            {
+                PageId = PageId,
+                Username = username,
+                ApiToken = apiToken,
+                ApiVersion = ApiVersion.V2,
+                ConfluenceDomainName = domainName
+            },
+            CancellationToken.None
+        );
+        Assert.AreEqual(200, result.StatusCode);
+    }
+
+    [TestMethod]
+    public async Task DeletePage()
+    {
+        var result = await Confluence.DeletePage(
+            new Input
+            {
+                PageId = PageId,
+                Username = username,
+                ApiToken = apiToken,
+                ApiVersion = ApiVersion.V2,
+                ConfluenceDomainName = domainName
+            },
+            CancellationToken.None
+        );
+        Assert.AreEqual(204, result.StatusCode);
+    }
+
+    [TestMethod]
+    public async Task GetPageByTitle()
+    {
+        var result = await Confluence.GetPageByTitle(
+            new Input
+            {
+                Title = "TestPage1",
+                SpaceKey = "TEST",
+                Username = username,
+                ApiToken = apiToken,
+                ApiVersion = ApiVersion.V2,
+                ConfluenceDomainName = domainName
+            },
+            CancellationToken.None
+        );
+        Assert.AreEqual(200, result.StatusCode);
+    }
+
+    [TestMethod]
+    public async Task CreateSpace()
+    {
+        var input = new Input
+        {
+            SpaceName = "Test123",
+            SpaceKey = "TEST2",
+            Username = username,
+            ApiToken = apiToken,
+            ApiVersion = ApiVersion.V1,
+            ConfluenceDomainName = domainName
+        };
+        await Confluence.DeleteSpace(input, CancellationToken.None);
+        var result = await Confluence.CreateSpace(input, CancellationToken.None);
+
+        Assert.AreEqual(200, result.StatusCode);
+
+        await Confluence.DeleteSpace(input, CancellationToken.None);
+    }
```

Comment:
It can't be NewGuid because it doesn't match the standards for naming spaces. The tests are constructed to delete the newly created space, so there won't be an issue with duplicate space keys and names.